### PR TITLE
ID-347 Fixed lighthouse contrast fails in bootstrap

### PR DESCRIPTION
**Pull request by WarwickGoatee**
Visual differences are minimal and lighthouse is no longer upset with us.

### CODE REVIEW

File: less/bootstrap-fixes.less [hunk 1]
```diff
@@ -213,4 +213,30 @@ label.checkbox, label.radio, label.checkbox-inline, label.radio-inline {
...
 .popover button.close {
   opacity: 1;
-}
\ No newline at end of file
+}
+
+/* Contrast fixes ID-347 */
+
+blockquote small {
+  color: @text-muted;
```
Comment (matmannion) [inline less/bootstrap-fixes.less]: Better to override `@blockquote-small-color` in variables.less so it just updates the Boostrap default, and also applies to any other rules it's used in

File: less/bootstrap-fixes.less [hunk 2]
```diff
@@ -213,4 +213,30 @@ label.checkbox, label.radio, label.checkbox-inline, label.radio-inline {
...
+}
+
+.help-block {
+  color: darken(@text-muted, 10%);
+}
+
+.nav > li.disabled > a, .pager .disabled > a {
+  color: @text-muted;
```
Comment (matmannion) [inline less/bootstrap-fixes.less]: `@nav-disabled-link-color`, `@pager-disabled-color`

File: less/bootstrap-fixes.less [hunk 3]
```diff
@@ -213,4 +213,30 @@ label.checkbox, label.radio, label.checkbox-inline, label.radio-inline {
...
+}
+
+.breadcrumb > .active {
+  color: darken(@text-muted, 10%);
```
Comment (matmannion) [inline less/bootstrap-fixes.less]: `@breadcrumb-active-color`

File: less/bootstrap-fixes.less [hunk 4]
```diff
@@ -213,4 +213,30 @@ label.checkbox, label.radio, label.checkbox-inline, label.radio-inline {
...
+}
+
+.badge {
+  background-color: @text-muted;
```
Comment (matmannion) [inline less/bootstrap-fixes.less]: `@badge-bg`

File: less/variables.less [hunk 5]
```diff
@@ -145,7 +145,7 @@
 @brand-primary: #204f79;
 @brand-success: #596e31;
 @brand-info: #1d817a;
-@brand-warning: lighten(@id7-brand-orange, 10%);
```
Comment (nanoamp) [inline less/variables.less]: I'm wondering if this is the best way to effect this change. There's not much discernible difference between warning and danger now.

File: less/bootstrap-fixes.less [hunk 6]
```diff
@@ -213,4 +213,30 @@ label.checkbox, label.radio, label.checkbox-inline, label.radio-inline {
...
+}
+
+.panel-warning > .panel-heading {
+  color: darken(@brand-warning, 10%);
```
Comment (matmannion) [inline less/bootstrap-fixes.less]: `@panel-warning-text`

Comment (WarwickGoatee) [reply less/bootstrap-fixes.less]: Change'd

**Comment by WarwickGoatee** [comment]
Checked with the content team and they're happy with the warning button colour.

**Comment by WarwickGoatee** [comment]
Version number up'd

**Pull request merged**